auth-web/src/components/auth/staff/review-task/QsApplication.vue

```diff
@@ -0,0 +1,227 @@
+      </v-col>
+    </v-row>
+
+    <!-- Authorization Information -->
+    <v-row>
+      <v-col class="cols-12 mt-5 py-2">
+        <h3>Authorization</h3>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="py-2">
+        Legal name of the person authorized to complete and submit this application. <b>Note:</b> The authorized person
+        must be an active B.C. lawyer or notary in good standing.
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="py-2 icon-text">
+        <v-icon class="pr-2" color="success">mdi-check</v-icon>
+        <span>
+          <b>{{ qsApplicantData && qsApplicantData.authorizationName }}</b> certifies that they have relevant knowledge
+          of the Qualified Supplier and is authorized to submit this application.
+        </span>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="py-0 ml-8">
+        <b>Date:</b> {{ formatDate(taskDetails.created) }}
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="py-2 ml-8 fs-14">
+        Note: It is an offence to make or assist in making a false or misleading statement in a record filed under the
+        Manufactured Home Act. A person who commits this offence is subject to fine of up to $2,000.
+      </v-col>
+    </v-row>
+  </section>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, onMounted, reactive, toRefs } from '@vue/composition-api'
+import BaseAddressForm from '@/components/auth/common/BaseAddressForm.vue'
+import CommonUtils from '@/util/common-util'
+import { Contact } from '@/models/contact'
+import { Organization } from '@/models/Organization'
+import { QualifiedSupplierApplicant } from '@/models/external'
+import { Task } from '@/models/Task'
+import TaskService from '@/services/task.services'
+import { User } from '@/models/user'
+import moment from 'moment/moment'
+import { userAccessRequirements } from '@/resources/QualifiedSupplierAccessResource'
+
+export default defineComponent({
+  name: 'QsApplication',
+  components: {
+    BaseAddressForm
+  },
+  props: {
+    tabNumber: { type: Number, default: null },
+    title: { type: String, default: 'Qualified Supplier Application' },
+    taskDetails: { type: Object as () => Task, default: () => null },
+    accountUnderReview: { type: Object as () => Organization, default: () => null },
+    accountUnderReviewAdmin: { type: Object as () => User, default: () => null },
+    accountUnderReviewAdminContact: { type: Object as () => Contact, default: () => null }
+  },
+  setup (props) {
+    const localState = reactive({
+      qsApplicantData: null as QualifiedSupplierApplicant,
+      qsApplicationTypeDisplay: computed(() => {
+        return props.taskDetails?.type.replace(/.*–\s*/, '')
+      }),
+      qsApplicantPhone: computed(() => {
+        return CommonUtils.toDisplayPhone(localState.qsApplicantData?.phoneNumber)
+      }),
+      qsRequirements: computed(() => {
```

Comment:
Cool I didn't know you could do this

Reply:
Heck yea!
Wrap the local state and cut way down on the properties in the return, just spread!